--- rgdal.orig/src/Makevars.win	2017-11-21 15:41:55.000000000 -0200
+++ rgdal/src/Makevars.win	2018-01-28 15:18:51.373650734 -0200
@@ -2,17 +2,10 @@
 RWINLIB = lib${subst gcc,,${COMPILED_BY}}${R_ARCH}
 
 PKG_CPPFLAGS =\
-	-I../windows/gdal2-2.2.0/include/gdal \
-	-I../windows/gdal2-2.2.0/include/geos \
-	-I../windows/gdal2-2.2.0/include/proj
+	-I$(LIBRARY_INC)
 
 PKG_LIBS = \
-	-L../windows/gdal2-2.2.0/${RWINLIB} \
-	-lgdal -lsqlite3 -lspatialite -lproj -lgeos_c -lgeos  \
-	-ljson-c -lnetcdf -lpq -lintl -lwebp -lcurl -lssh2 -lssl -lcrypto \
-	-lhdf5_hl -lhdf5 -lexpat -lfreexl -lcfitsio \
-	-lpng16 -ljpeg -ltiff -lgeotiff -lgif -lxml2 -llzma -lszip -lz \
-	-lodbc32 -lodbccp32 -liconv -lpsapi -lws2_32 -lcrypt32 -lwldap32 -lsecur32 -lgdi32
+	-L$(LIBRARY_LIB)
 
 all: clean winlibs
 
@@ -21,8 +14,7 @@
 winlibs:
 	mkdir -p ../inst
 	"${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" --vanilla "../tools/winlibs.R"
-	cp -r ../windows/gdal2-2.2.0/share/gdal ../inst/
-	cp -r ../windows/gdal2-2.2.0/share/proj ../inst/
+	cp -r $(LIBRARY_PREFIX)/share/gdal ../inst/
 
 clean:
 	rm -f $(SHLIB) $(OBJECTS)
